README.md

```diff
@@ -62,8 +71,14 @@ resource "aws_wafregional_ipset" "global" {
   }
 
 }
-...
-  ip_set = aws_wafregional_ipset.global.id
 ```
 
 Use `terraform state mv` to externalize the IP Set, e.g., `terraform state mv FOO.BAR.aws_wafregional_ipset.ips Foo.aws_wafregional_ipset.ips`.
+
+### 1.3.0 to 2.0.0
+
+Version `2.0.0` removes the `environment` variable and adds `web_acl_metric_name` and `web_acl_name` variables to provide more control over naming.  AWS WAF rules will be prefixed by the `web_acl_name` of their associated Web ACL to provide for easy visual sorting.
+
+Version `2.0.0` replaces the `ip_set` variable with a `ip_sets` list variable, which accepts a list of `aws_wafregional_ipset` ids.  This variable allows the Web ACL to pull from multiple lists of blocked ip addresses, such that you can combine a global blocked list, and application-specific lists.  For example: `ip_sets = list(resource.aws_wafregional_ipset.global.id, resource.aws_wafregional_ipset.helloworld.id)`.
+
+During the initial upgrade to `2.0.0`, and if you add additional dynamic rules, you'll need to delete your web ACLs, as terraform cannot properly handle peer dependencies between Rules and Web ACLs.  For convenience, you can use the `delete-web-acl` script in the scripts folder.  For example: `scripts/delete-web-acl WEB_ACL_ID`.  Once the Web ACL is deleted use terraform apply to recreate the Web ACL and associate with your resources as you had before.  Deleting a Web ACL does not delete any associated resources, such as Application Load Balancers; however, it will lead the resources temporarily unprotected.
```

Comment:
```suggestion
During the initial upgrade to `2.0.0`, and if you add additional dynamic rules, you'll need to delete your web ACLs, as terraform cannot properly handle peer dependencies between Rules and Web ACLs.  For convenience, you can use the `delete-web-acl` script in the scripts folder.  For example: `scripts/delete-web-acl WEB_ACL_ID`.  Once the Web ACL is deleted use terraform apply to recreate the Web ACL and associate with your resources as you had before.  Deleting a Web ACL does not delete any associated resources, such as Application Load Balancers; however, it will leave the resources temporarily unprotected.
```

Reply:
`s/lead/leave/g`